prov/rxm/src/rxm_ep.c

```diff
@@ -421,12 +421,12 @@ static int rxm_ep_recv_common(struct rxm_ep *rxm_ep, const struct iovec *iov,
 			dlist_insert_tail(&recv_entry->entry,
 					  &recv_queue->recv_list);
 			ret = 0;
+			fastlock_release(&recv_queue->lock);
```

Comment:
I saw the coverity issue, so I agree that something is needed., but I'm not sure this is correct.  This would leave the freestack_push() below unlocked.